nixos/modules/services/networking/iodine.nix

```diff
@@ -9,6 +9,8 @@ let
 
   iodinedUser = "iodined";
 
+  /* is this path made unreadable by ProtectHome = true ? */
+  isProtected = x: hasPrefix "/root" x || hasPrefix "/home" x;
```

Comment:
`ProtectHome=yes` makes `/home`, `/root`, and `/run/user` unrechable. I'd propose moving this much closer to the single place where it's used, or maybe just inlining it entirely (it's not thaat long)

Reply:
It's used in two places, so I would prefer not in-lining it.

Reply:
ok.